examples/puts3_config.yml

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MiNiFi Config Version: 3
+Flow Controller:
+  name: MiNiFi Flow
+Processors:
+- Properties:
+    Input Directory: /tmp/input
+  auto-terminated relationships list: []
+  class: org.apache.nifi.processors.standard.GetFile
+  id: bfd376a8-c845-453b-9f74-4cf6f3a95e74
+  name: GetFile
+  penalization period: 30 sec
+  run duration nanos: 0
+  scheduling period: 2 sec
+  scheduling strategy: TIMER_DRIVEN
+  yield period: 1 sec
+- Properties:
+    Access Key: test_access_key
+    Bucket: test_bucket
+    Secret Key: test_secret
+  auto-terminated relationships list:
+  - failure
+  class: org.apache.nifi.processors.standard.PutS3Object
+  id: 8c9e2512-e814-4b73-ac5c-f33753332128
+  name: PutS3Object
+  penalization period: 30 sec
+  run duration nanos: 0
+  scheduling period: 1 sec
+  scheduling strategy: TIMER_DRIVEN
+  yield period: 1 sec
+- Properties:
+    Directory: /tmp/output
+    Directory Permissions: '777'
+    Permissions: '777'
+  auto-terminated relationships list:
+  - success
+  - failure
+  class: org.apache.nifi.processors.standard.PutFile
+  id: d7dbc3af-ee24-4bc0-8c63-90e0f3185e0f
+  name: PutFile
+  penalization period: 30 sec
+  run duration nanos: 0
+  scheduling period: 1 sec
+  scheduling strategy: EVENT_DRIVEN
+  yield period: 1 sec
+Connections:
+- destination id: 8c9e2512-e814-4b73-ac5c-f33753332128
+  drop empty: 'false'
+  name: cc62a4a5-3a43-4725-8d05-d0b3e65d1ee6
```

Comment:
More descriptive names are a good practice. For connections, the usual source-name/relationship/destination-name is fine. For processors, something like "Upload to S3 test_bucket" is preferrable over "PutS3Object".

Reply:
Good point, especially for newcomers for easier understanding. Fixed in 369f1085d136a749731b067dad19c73cded52532